hack/tools.go

```diff
@@ -21,6 +21,7 @@ package tools
 
 // This package imports things required by this repository, to force `go mod` to see them as dependencies
 import (
+	_ "github.com/kcp-dev/code-generator"
```

Comment:
can't do this until https://github.com/kcp-dev/code-generator/pull/32 makes it in